deployments/unichain/usdc/roots.json

```diff
@@ -1,12 +1,12 @@
 {
-  "comet": "0x2c7118c4C88B9841FCF839074c26Ae8f035f2921",
-  "configurator": "0x8df378453Ff9dEFFa513367CDF9b3B53726303e9",
-  "rewards": "0x6f7D514bbD4aFf3BcD1140B7344b32f063dEe486",
-  "cometFactory": "0xdB7EdFa090061D9367CbEAF6bE16ECbDE596676C",
-  "bridgeReceiver": "0x4b5DeE60531a72C1264319Ec6A22678a4D0C8118",
-  "l2CrossDomainMessenger": "0x4200000000000000000000000000000000000007",
-  "l2StandardBridge": "0x4200000000000000000000000000000000000010",
-  "bulker": "0x58EbB8Db8b4FdF2dCbbB16E04c2F5b952963B514",
-  "COMP": "0xdf78e4f0a8279942ca68046476919a90f2288656",
-  "TokenMinter": "0x726bFEF3cBb3f8AF7d8CB141E78F86Ae43C34163"
+    "comet": "0x2c7118c4C88B9841FCF839074c26Ae8f035f2921",
+    "configurator": "0x8df378453Ff9dEFFa513367CDF9b3B53726303e9",
+    "rewards": "0x6f7D514bbD4aFf3BcD1140B7344b32f063dEe486",
+    "cometFactory": "0xdB7EdFa090061D9367CbEAF6bE16ECbDE596676C",
+    "bridgeReceiver": "0x4b5DeE60531a72C1264319Ec6A22678a4D0C8118",
+    "l2CrossDomainMessenger": "0x4200000000000000000000000000000000000007",
+    "l2StandardBridge": "0x4200000000000000000000000000000000000010",
+    "bulker": "0x58EbB8Db8b4FdF2dCbbB16E04c2F5b952963B514",
+    "COMP": "0xdf78e4f0a8279942ca68046476919a90f2288656",
+    "TokenMinter": "0x726bFEF3cBb3f8AF7d8CB141E78F86Ae43C34163"
```

Comment:
We probably don't need these formatted with extra spaces. We removed them in the above roots.